test.env

```diff
@@ -0,0 +1,10 @@
+FACEBOOK_APP_ID=1234567890123456
+FACEBOOK_PAGE_ID=1029384756
+MESSENGER_APP_SECRET=0123456789abcdef0123456789abcdef
+MESSENGER_VALIDATION_TOKEN=validate_me
+MESSENGER_PAGE_ACCESS_TOKEN=ThatsAReallyLongStringYouGotThere
+SERVER_URL=https://localhost/
+SLACK_WEBHOOK_URL=https://hooks.slack.com/services/T123456/B33Z/Pajamas
+SLACK_CHANNEL=channel
+LOG_FILE=/var/log/my-bot/chat.log
+ALLOW_CONFIG_MUTATIONS=Y
```

Comment:
`ALLOW_CONFIG_MUTATIONS` is the new config I needed. This will let you do `PORT=-1` in #54 without potentially confusing users.